test/cypress/integration/consumer-tools/rural-or-underserved-tool/rural-or-underserved-tool.js

```diff
@@ -6,9 +6,10 @@ describe( 'Rural or Underserved Tool', () => {
 
   describe( 'Address Search', () => {
 
-    it( 'Should render a table for a series of addresses searched', () => {
+    it( 'Should render a table for a single address', () => {
       page.open();
       page.searchAddress( '1600 Pennsylvania Avenue Washington DC' );
+      page.interceptCensusAPIRequests();
```

Comment:
All the `.as(…)` calls could be waited for in the tests where the API call is expected. E.g. something like:

```
page.open();
page.interceptCensusAPIRequests();
page.searchAddress( '1600 Pennsylvania Avenue Washington DC' );
// wait for API calls…
cy.wait('@censusGeocoding')
cy.wait('@censusTigerweb1')
cy.wait('@censusTigerweb2')
cy.wait('@censusTigerweb3')
// Or whatever order the calls happen in.
```

Reply:
@anselmbradford, I was considering waiting like that, but I wasn't sure about our general approach to using `cy.wait`. There are some waits scattered here and there throughout our tests, so it doesn't seem like we've had a consistent way of waiting before. In this case, since [Cypress says stubbed responses usually return in 20ms or so](https://docs.cypress.io/guides/guides/network-requests#Stub-Responses), I didn't put in explicit waiting; all four of those responses should come back well within the standard wait time. But I'm happy to add them in if we want to move to explicit waiting like that in our tests.

Reply:
As I understand it waiting for a request call is fine, and would test that the requests are happening at the right times. Waiting for an arbitrary amount of time is an anti-pattern, but that's not what this is. Generally if I have been intercepting a request and giving it a name, I would expect to wait for it somewhere else (otherwise, why does it have a name?)

Reply:
@anselmbradford, I added the waits in 1153b19, although I put them in the helper instead of the main test to keep all the intercept stuff together (speaking of things we do differently among all the tests, we're all over the place on what goes into a helper or not).

Even if we aren't waiting for intercepted requests, I still like giving them an alias with `.as()` to get the nice little pills showing up in the Cypress UI's test body output, like in the screenshot above.

Reply:
> Even if we aren't waiting for intercepted requests, I still like giving them an alias with .as() to get the nice little pills showing up in the Cypress UI's test body output, like in the screenshot above.

Ah good point!